src/DumpDataPartial.php

```diff
@@ -0,0 +1,102 @@
+<?php declare(strict_types=1);
+namespace DiabloMedia\Robo\Task\MysqldumpPhp;
+
+use Robo\Common\BuilderAwareTrait;
+use Robo\Contract\BuilderAwareInterface;
+use Robo\Result;
+
+class DumpDataPartial extends Dump implements BuilderAwareInterface
+{
+    use BuilderAwareTrait;
+
+    protected $tableFilters         = [];
```

Comment:
It's generally helpful to add types (through doc-blocks) to all properties too, for phpstan's analysis.